tests/out/quad-vert.Vertex.glsl

```diff
@@ -21,7 +21,7 @@ struct gen_gl_PerVertex_block_0 {
 vec2 a_pos = vec2(0, 0);
 layout(location = 1) in vec2 _p2vs_location1;
 layout(location = 0) in vec2 _p2vs_location0;
-smooth layout(location = 0) out vec2 _vs2fs_location0;
+smooth out vec2 _vs2fs_location0;
```

Comment:
I'm confused here. Why are we removing the output locations from these varyings now?

Reply:
This is not supported under webgl. According to this wiki, it requires at least version 4.1 and `ARB_separate_shader_objects`. I'm not sure if we care about that right now. Varying locations are not required when you always link all the stages together.